source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Views/Evaluations/Calification.cshtml

```diff
@@ -150,7 +150,10 @@
     </tbody>
   </table>
   <div class="footer-evaluation" data-bind="if: isEvaluationEditable()">
-    <button class="button-evaluations primary-btn finish-btn" data-bind="click: onFinish">Finalizar</button>
+    <button class="button-evaluations primary-btn finish-btn" data-bind="click: onFinish">
+      <span data-bind="if: isResposibleCalificating()"> finalizar e iniciar evaluación empresa </span>
+      <span data-bind="if: !isResposibleCalificating()"> finalizar</span>
```

Comment:
I am not sure if to add an `span` is a good idea, what do you think about to use _comment tags_?

For example:

```html
<ul>
    <li>This item always appears</li>
    <!-- ko if: someExpressionGoesHere -->
        <li>I want to make this item present/absent dynamically</li>
    <!-- /ko -->
</ul>
```

Also, what is the meaning of the space just before the word "finalizar"?